Entity/CategoryManager.php

```diff
@@ -127,7 +127,7 @@ public function getRootCategoryWithChildren(CategoryInterface $category)
     /**
      * @todo NEXT_MAJOR: remove this method.
      *
-     * @param ContextInterface $context
+     * @param ContextInterface|string|null $context
```

Comment:
Instead of supporting different types (objects and strings), could you please pick one and deprecate the other?